CMakePresets.json

```
@@ -56,6 +56,16 @@
         "THRUST_MULTICONFIG_ENABLE_SYSTEM_TBB": true
       }
     },
+    {
+      "name": "all-dev-debug",
+      "displayName": "all-dev debug",
+      "inherits": "all-dev",
+      "cacheVariables":{
```

Nit: could use a space here
```suggestion
      "cacheVariables": {
```

---

D'oh, I just saw this now, after the automerge happened. Agreed, I'll fix this as part of #1654 👍